### PR TITLE
Return non zero exit code on errors for clean-svg-ids

### PR DESCRIPTION
**Pull request by kesara**

### CODE REVIEW

**Comment by dkg** [comment]
@kesara can you give an example of a document where it's important to bubble up this failure as a critical failure?

**Comment by kesara** [comment]
> @kesara can you give an example of a document where it's important to bubble up this failure as a critical failure?

@dkg I don't have an example in hand.
IIRC this was an issue I came across when I was implementing https://author-tools.ietf.org/clean-svg-ids
Author Tools logic relied on exit codes to identify errors.

File: bin/kramdown-rfc-clean-svg-ids [hunk 1]
```diff
@@ -18,6 +18,7 @@ def svg_clean_ids(s)
 rescue => detail
   warn "*** Can't clean SVG: #{detail}"
   d.to_s
+  exit 1
```
Comment (cabo) [inline bin/kramdown-rfc-clean-svg-ids:21]: This means there will be no output while the previous line was careful preparing to return something useful.

I'm not even sure that failing in this case warrants an exit code 1.

Comment (kesara) [reply bin/kramdown-rfc-clean-svg-ids:21]: Yeah, it makes sense not to exit with `1`.

**Pull request closed without merging**